CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ CHANGELOG
 
 3.0.0
 ------
+**ENHANCEMENTS**
+- export `ParallelClusterApiInvokeUrl` so it can be used cross-stack
```

Comment:
this should go to a 3.x.x changelog entry

Reply:
Updated!